airflow/dags/rt_views/gtfs_rt_thread_errors.sql

```diff
@@ -0,0 +1,100 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.gtfs_rt_thread_errors"
+
+description: | Each row is a unique file_hash of a feed that generated an exception within GCP logger
+
+
+fields:
+  file_hash: unique identifier for each feed download attempt.
+  calitp_itp_id: Feed ITP ID.
+  calitp_url_number: Feed URL number.
+  feed_file: Combination calitp_itp_id, URL number, and feed URL name.
+  feed_file_name: feed url name.
+  start_fetch_time: timestamp of start_fetch thread as logged by archiver.
+  completed_fetch_time: timestamp of completed_fetch thread as logged by archiver.
+  start_write_time: timestamp of start_write thread as logged by archiver.
+  completed_write_time: timestamp of completed_write thread as logged by archiver.
+  error_time: timestamp when thread exception occured.
+  error_message: error message contained within the exception.
+---
+
+-- note that archiver will log a completed write time regardless if fetch was successful due to them being on separate threads
+-- this will be changed in future iterations
+
+With
+start_fetch AS (
+    SELECT
+        textpayload,
+        timestamp AS start_fetch_time,
+        REGEXP_EXTRACT(textPayload, r'\[txn (.*?)\]') AS file_hash,
```

Comment:
should this column be called `transaction_id`? I think it's a generated uuid, not a hash of anything.

Also do you need a lazy evaluation here? `)` The UUID won't contain a bracket character.

Reply:
good point, I had heard it referred to as a hash but it isn't actually a hash. Will rename it across both tables